recipe/0001-Add-TABDATA-to-install_data.patch

```diff
@@ -0,0 +1,14 @@
+--- Makefile
++++ Makefile
+@@ -679,9 +679,10 @@
+  ZIC='$(ZIC)'
+ 
+-INSTALL_DATA_DEPS = zic leapseconds yearistype tzdata.zi
++INSTALL_DATA_DEPS = zic leapseconds yearistype tzdata.zi $(TABDATA)
```

Comment:
Why do you need to patch this?

Reply:
Just to avoid the manual copying of those extra files ;).
But if we don't need them, we can remove that whole commit. I just don't know if it makes sense to include them or not.
(ref: https://github.com/python/tzdata/issues/28)

Reply:
I'm not sure why you have to patch anything to get those files. Upstream the way I unpack this is [here](https://github.com/python/tzdata/blob/842c5384d0a02c30de513b33277369d488f374c2/update.py#L86-L99). I do:

`make DESTDIR={temporary_directory} POSIXRULES= ZFLAGS='-b slim' install`

(You would use `-b fat`) and then move everything that's in `temporary_directory/usr/share/zoneinfo` over to the location I'm expecting. As far as I know no additional copying is necessary.

Reply:
I just wanted a Make target that installed everything in `TZDIR`. But I'm also more than fine with the way you did it and added a commit to do just that!
The tests in this recipe will also catch when anything unexpected would get installed, so all good either way.